test/e2e/framework/util.go

```diff
@@ -4270,8 +4271,7 @@ func GetNodePortURL(client clientset.Interface, ns, name string, svcPort int) (s
 	return "", fmt.Errorf("Failed to find external address for service %v", name)
 }
 
-// TODO(random-liu): Change this to be a member function of the framework.
-func GetPodLogs(c clientset.Interface, namespace, podName, containerName string) (string, error) {
+func (f *Framework) GetPodLogs(c clientset.Interface, namespace, podName, containerName string) (string, error) {
```

Comment:
Presumably the implication of the comment is that you should remove the clientset and namespace arguments and get them from `f` directly. Otherwise there's no advantage to having the function be a member function

Reply:
Agree.